addons/mail/views/assets.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <template id="_assets_primary_variables" inherit_id="web._assets_primary_variables">
```

Comment:
Need to create a migration branch due to removed record `mail._assets_primary_variables`. I think one line with proper call to `remove_view` helper should do it?

Reply:
You spoke Chinese for me ... Never did any migration so ..

Reply:
This is a good entry point: https://github.com/odoo/upgrade/wiki/How-To 
Then feel free to ask questions

Reply:
Not sure if it has to be done in pre-migration or another hook but I did in pre-migration : https://github.com/odoo/upgrade/pull/1421

Reply:
I don't understand why it failed :

```
odoo.addons.base.maintenance.migrations.base.pre-models-ir_ui_view:211 unlink It's look like you forgot to call `util.remove_view(cr, 'mail._assets_helpers')`
--
```

It tells me to delete the one we added ... ? looks like he's trying to migrate back ... So should we bump a version for discuss and create a script for saas-13.4.1.1 or something like that ?

Reply:
As said, you can ignore that error for now, this is only due to us doing a migration in not master, which is not supposed to be allowed.